pkg/liqonet/tunnel/wireguard/driver.go

```diff
@@ -52,6 +52,9 @@ const (
 	KeepAliveInterval = 10 * time.Second
 	// MTU size of mtu for wireguard interface.
 	MTU = 1415
+	// Maybe we can remove privileged ones?
```

Comment:
Can we?

Reply:
The vpn endpoint is exposed using a `NodePort` or a `LoadBalancer`. By default the port range is in 30000-32767. On the other hand some LoadBalancer implementations could expose a given service using a node port or directly  routing traffic to the backend pods. Being the port range configurable I just check the port to be inside the valid range of possible UDP ports.

Reply:
Fine by me :+1: . But a question in a comment is quite misleading.